game/entities/sdBullet.js

```diff
@@ -57,10 +57,38 @@ class sdBullet extends sdEntity
 		
 		sdWorld.entity_classes[ this.name ] = this; // Register for object spawn
 	}
-	get hitbox_x1() { return this.is_grenade || this.ac > 0 ? -2 : -0.1; }
-	get hitbox_x2() { return this.is_grenade || this.ac > 0 ? 2 : 0.1; }
-	get hitbox_y1() { return this.is_grenade || this.ac > 0 ? -2 : -0.1; }
-	get hitbox_y2() { return this.is_grenade || this.ac > 0 ? 2 : 0.1; }
+	get hitbox_x1()
+	{ if ( this.model_is_large ) // if bullet sprite model is 64 by 64
```

Comment:
We probably should replace all these extra properties with single property that tells describes radius. Each of these public properties are being sent to players on each sync - kind of too much data to send.